File: strings/389. Find the Difference/findTheDifference.js
/**
 * @param {string} s
 * @param {string} t
 * @return {character}
 */
var findTheDifference = function(s, t) {
    for (let char of t) {
        let countS = 0;
        let countT = 0;

        // Count occurrence of char in s
        for (let j = 0; j < s.length; j++) {
            if (s[j] === char) {
                countS++;
            }
        }

        // Count occurrence of char in t
        for (let j = 0; j < t.length; j++) {
            if (t[j] === char) {
                countT++;
            }
        }

        if (countT > countS) {
            return char;
        }
    }
};

File: strings/389. Find the Difference/findTheDifference.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        for char in t:
            if char not in s or t.count(char) > s.count(char):
                return char